### ISSUE TITLE
click on ListAddField removes already defined values

### ISSUE THREAD

**Issue by macrozone** [vazco/uniforms#128]
steps to reproduce:

- run demo,
- define a list-field: 

```
{
   emails: {type: Array},
   "emails.$": {type: String}

}
```

- click on plus
- add a first value
- click again on plus

There are now two empty fields instead of the first containing the value.

I think the error lies in how the add fields receives the value from the parent. It uses the `includeParent` from `connectField`. But this seems to copy the parent-props instead of passing a reference.

Edit: a similar issue happens also with the ListDelField.

**Comment by radekmie** [comment]
Which version? It was fixed recently in v1.4.1 due to #125.

**Comment by macrozone** [comment]
it's working properly with 1.5.0 <3 <3 <3
